angular/src/app/search/search.component.html

```diff
@@ -28,7 +28,8 @@
     </mat-panel-description>
   </mat-expansion-panel-header>
   <mat-list>
-    <mat-list-item>
+    <!-- Use when tags get implimented -->
+    <!-- <mat-list-item>
```

Comment:
I'll let this chunk of comments slide for now since I'll use it very soon, but we shouldn't be having commented out code.

Reply:
Thanks Daniel, I'll remember that for next time.